Content.Client/Silicons/Borgs/BorgSwitchableTypeSystem.cs

```diff
@@ -40,8 +45,14 @@ protected override void UpdateEntityAppearance(
     {
         if (TryComp(entity, out SpriteComponent? sprite))
         {
-            _sprite.LayerSetRsiState((entity, sprite), BorgVisualLayers.Body, prototype.SpriteBodyState);
-            _sprite.LayerSetRsiState((entity, sprite), BorgVisualLayers.LightStatus, prototype.SpriteToggleLightState);
```

Comment:
The old code here deals with the light while the new code does not. Do borg lights still work?